lib/Controller/ChatController.php

```diff
@@ -790,6 +795,10 @@ public function getObjectsSharedInRoomOverview(int $limit = 7): DataResponse {
 		$comments = $this->chatManager->getMessagesById($this->room, array_merge(...array_values($messageIdsByType)));
 
 		foreach ($comments as $comment) {
+			if (!$this->messageParser->fileOfMessageExists($comment->getMessage())) {
```

Comment:
```suggestion
			if ($this->messageParser->isSharedFile($comment->getMessage()) && !$this->messageParser->fileOfMessageExists($comment->getMessage())) {
```

Reply:
Won't we have only messages as shared file here?

Reply:
No, it's also contacts, polls, locations, ....